src/hooks.tsx

```diff
@@ -278,8 +277,8 @@ export const useTable = <T extends DataType>(
       perPage: 10,
       canNext: true,
       canPrev: false,
-      nextPage: () => {},
-      prevPage: () => {},
+      nextPage: /* istanbul ignore next */ () => {},
```

Comment:
Not sure how I feel about having these comments everywhere but I'm okay with it for now.